apps/coscrad-frontend/src/components/resources/books/book-detail.full-view.presenter.tsx

```diff
@@ -10,26 +15,50 @@ export const BookDetailFullViewPresenter = ({
     author,
     publicationDate,
 }: ICategorizableDetailQueryResult<IBookViewModel>): JSX.Element => {
+    const fullScreenHandle = useFullScreenHandle();
+    const [screenState, setScreenState] = useState(false);
```

Comment:
It's better to name booleans using `isX`, `doesX`, `hasX`.  I'd use something like, `isInFullScreenMode` (`setIsInFullScreenMode`).